packages/components/src/font-size-picker/index.js

```diff
@@ -26,76 +31,61 @@ function FontSizePicker( {
 	value,
 	withSlider = false,
 } ) {
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
+	const [ currentSelectValue, setCurrentSelectValue ] = useState( getInitialSelectValue( fontSizes, value ) );
```

Comment:
It seems like this changes the behavior of the control. It no longer will get its value updated when someone manually enters the value in the input. The idea is that this combo box should also update when someone picks the font size which matches one of the presets.

Reply:
Oh, that functionality is working for me locally. It doesn't update if you empty the input box, but if you add any value to it the select updates to the correct option. Could you give me more details about what you tried so I can try to reproduce it? Thanks!

Reply:
In `master` branch when you reset value using the reset button or by emptying the input box it reflects with the font size update to the default name – Normal – in case of Paragraph block. It doesn't work this way in this branch. There is one source of truth in the previous implementation – `value` – which can be updated with `onChange` helper. In this branch, there is a local state introduced which shadows the same value but in my testing, it doesn't fully replicate the old behavior. This component updated every time `value` changes, so it might be enough to keep it working similar to what existed before.

Reply:
It should be following previous behaviour exactly now!